src/web/layouts/StandardLayout.tsx

```diff
@@ -427,6 +430,17 @@ export const StandardLayout = ({ CAPI, NAV }: Props) => {
 
                     {showOnwardsLower && <Section islandId="onwards-lower" />}
 
+                    {showComments && (
+                        <Section>
+                            <Flex>
+                                <div id="comments-root" />
+                                <RightColumn>
+                                    <p>TODO: Comments ad slot goes here</p>
```

Comment:
do we want to display this text? or just have `// TODO:`?

Reply:
You're right, that p tag is dangerous. I'd much rather a mistake exposed a blank section rather than this text!